content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -347,41 +350,47 @@ rules:
   verbs: ["get", "list", "watch"]
 ```
 
-Allow "GET" and "POST" requests to the non-resource endpoint "/healthz" and all subpaths (must be in a `ClusterRole` bound with a `ClusterRoleBinding` to be effective):
+Allow GET and POST requests to the non-resource endpoint `/healthz` and all subpaths (must be in a ClusterRole bound with a ClusterRoleBinding to be effective):
 
 ```yaml
 rules:
 - nonResourceURLs: ["/healthz", "/healthz/*"] # '*' in a nonResourceURL is a suffix glob match
   verbs: ["get", "post"]
 ```
 
-### Referring to Subjects
+### Referring to subjects
 
 A RoleBinding or ClusterRoleBinding binds a role to subjects.
 Subjects can be groups, users or
 {{< glossary_tooltip text="ServiceAccounts" term_id="service-account" >}}.
 
-Users are represented by strings.  These can be plain usernames, like
-"alice", email-style names, like "bob@example.com", or numeric IDs
-represented as a string.  It is up to the Kubernetes admin to configure
-the [authentication modules](/docs/reference/access-authn-authz/authentication/) to produce
-usernames in the desired format.  The RBAC authorization system does
-not require any particular format.  However, the prefix `system:` is
-reserved for Kubernetes system use, and so the admin should ensure
-usernames do not contain this prefix by accident.
+Kubernetes RBAC represents usernames represented as strings.
+These can be: plain usernames, such as "alice"; email-style names, like "bob@example.com";
+or numeric user IDs represented as a string.  It is up to you as a cluster administrator
+to configure the [authentication modules](/docs/reference/access-authn-authz/authentication/)
+so that authentication produces usernames in the format you want.
+
+{{< caution >}}
+The prefix `system:` is reserved for Kubernetes system use, so you should ensure
+that no users or groups have names that start with `system:`
```

Comment:
The previous text was more accurate. Some system users/groups do contain this prefix. Cluster administrators should ensure that no non-system authentication provider uses this prefix.